src/odemis/driver/test/technolution_test.py

```diff
@@ -1420,6 +1454,47 @@ def test_get_field_and_mega_field_combination(self):
         self.assertEqual(field_images[0] * field_images[1], self.counter)
         self.assertEqual(field_images[0] * field_images[1], self.counter2)
 
+class Test_AsmApiException(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        if TEST_NOHW:
+            raise unittest.SkipTest('No simulator for the ASM and HwComponents present. Skipping tests.')
```

Comment:
This can also be the case if no real HW is present.
`No HW or simulator present. Skip tests. `